helm/qontract-reconcile/templates/template.yaml

```diff
@@ -361,8 +361,8 @@ objects:
   spec:
     schedule: "{{ $integration.cron }}"
     concurrencyPolicy: {{ $integration.concurrencyPolicy | default "Allow" }}
-    successfulJobHistoryLimit: {{ $integration.successfulJobHistoryLimit | default "3" }}
-    failedJobHistoryLimit: {{ if not (kindIs "invalid" $integration.failedJobHistoryLimit) }}{{ $integration.failedJobHistoryLimit }}{{ else }}{{ 1 }}{{ end }}
+    successfulJobsHistoryLimit: {{ $integration.successfulJobHistoryLimit | default "3" }}
```

Comment:
Doesn't this have the same typing issue as `failedJobsHistoryLimit`? Shouldn't this have an integer as its default?

Reply:
did the same as #2400 in https://github.com/app-sre/qontract-reconcile/pull/2402/commits/92b66cf7f29abdfef8f97e358299ed6f963acbf4

Reply:
the issue doesn't exist because no integration is using `0` :)